documentation/docs/documentation/06-generators/openapi/01-introduction.mdx

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: Openapi generator introduction
+sidebar_label: Introduction
+---
+
+Note: Openapi Hyper Fetch code generator is still in beta. Please let us know in case of any problems.
+
+Note: Right now this code generator accepts only V3 version of the openapi schema.
+
```

Comment:
Use the mdx admonitions

::: info

Openapi Hyper Fetch code generator is still in beta. Please let us know in case of any problems.

:::